src/Camera.cc

```diff
@@ -3,7 +3,7 @@
 #include "printutils.h"
 
 Camera::Camera() :
-	projection(ProjectionType::PERSPECTIVE), fov(22.5), viewall(false)
+	projection(ProjectionType::PERSPECTIVE), fov(22.5), viewall(false), autocenter(false), height(0.0)
```

Comment:
`height` doesn't seem to be used anywhere

Reply:
Oh, oops, indeed. Removed now.